Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM centos:7
```

Comment:
I think I messed up naming originally.

This file should be named `Dockerfile.builder` as it builds the binaries in docker container, and the current `Dockerfile.builder` we can name `Dockerfile.prow`. WDYT?

This will impact original `Makefile` too

Reply:
make sense. I have changed the names several times - I had also:
`Dockerfile.builder` for this one
`Dockerfile.binary` for the prow binaries, but your name is fine too

Reply:
Yeah, I think we can make sure we have consistent naming with other projects, so we won't have confusion. Thanks!

Reply:
renamed 87811af